README.md

```diff
@@ -622,88 +632,231 @@ There are some settings that should be reviewed and updated as needeed in the se
+after deployment. Alternatively, you can set credentials directly via `OPENSEARCH_USERNAME`
+and `OPENSEARCH_PASSWORD` environment variables instead of using Secrets Manager.
+
+**3. IAM permission to access secrets:**
+
+```yaml
+iam:
+  role:
+    statements:
+      - Effect: Allow
+        Resource: arn:aws:secretsmanager:${aws:region}:${aws:accountId}:secret:${self:provider.environment.OPENSEARCH_CREDENTIALS_SECRET_ID}-*
+        Action: secretsmanager:GetSecretValue
+```
 
-After reviewing the settings, build and deploy:
+**Option B: AWS IAM Authentication**
+
+Modify your serverless.yml file as follows:
+
+**1. Remove or comment out OpenSearch credential environment variables:**
+
+```yaml
+environment:
+  # OPENSEARCH_CREDENTIALS_SECRET_ID: ${self:service}-${self:provider.stage}-opensearch-user-creds
+```
+
+**2. Remove or comment out Secrets Manager IAM permission:**
+
+```yaml
+iam:
+  role:
+    statements:
+      # ... other statements
+      # Not needed for IAM authentication - only for fine-grained access control
+      # - Effect: Allow
+      #   Resource: arn:aws:secretsmanager:${aws:region}:${aws:accountId}:secret:${self:provider.environment.OPENSEARCH_CREDENTIALS_SECRET_ID}-*
+      #   Action: secretsmanager:GetSecretValue
+```
+
+**3. Disable fine-grained access control in OpenSearch domain:**
+
+```yaml
+OpenSearchInstance:
+  Type: AWS::OpenSearchService::Domain
+  Properties:
+    # ... other properties
+    AdvancedSecurityOptions:
+      Enabled: false
+      InternalUserDatabaseEnabled: false
+      # MasterUserOptions not needed for IAM authentication
+      #   MasterUserName: admin
+      #   MasterUserPassword: ${env:OPENSEARCH_MASTER_USER_PASSWORD}
+```
+
+**4. Ensure Lambda has OpenSearch permissions (already in default config):**
+
+```yaml
+iam:
+  role:
+    statements:
+      - Effect: Allow
+        Resource: "arn:aws:es:${aws:region}:${aws:accountId}:domain/*"
+        Action: "es:*"
+```
+
+**5. Configure access policy for OpenSearch domain:**
+
+When using IAM authentication without fine-grained access control, AWS requires that you
+restrict access to specific AWS accounts (you cannot use the wildcard `"AWS": "*"`, which
+allows any AWS account). Use the account root principal to restrict access to your AWS
+account only:
+
+```yaml
+OpenSearchInstance:
+  Type: AWS::OpenSearchService::Domain
+  Properties:
+    # ... other properties
+    AccessPolicies:
+      Version: "2012-10-17"
+      Statement:
+        - Effect: "Allow"
+          Principal: { "AWS": "arn:aws:iam::${aws:accountId}:root" }
+          Action: "es:ESHttp*"
+          Resource: "arn:aws:es:${aws:region}:${aws:accountId}:domain/${self:service}-${self:provider.stage}/*"
+```
+
+This restricts access to IAM principals in your AWS account only. Within your account,
+access is further controlled by the IAM permissions configured above (only principals with
+`es:*` can access).
+
```

Comment:
Not sure where best this fits in the flow, but it feels like `STAC_API_URL` should be commented out in the serverless.example.yml.  That way things can be tested.  The default value of `"https://some-stac-server.example.com"` makes for broken links in testing the API.

Maybe populated and uncommented with the cloudfront section of this documentation?